Mock-Get-Started/Mock-Get-Started-MyApp/src/main/java/com/microej/example/mock/MySNI.java

```diff
@@ -39,6 +39,12 @@ public static void sendText(String text) {
 		sendText(text.length());
 	}
 
+	/**
+	 * Does nothing
+	 */
+	public static void doNothing() {
+
+	}
```

Comment:
ok, this is weird.

a method that does nothing has no reason to be used and can be removed, there's probably a reason why you need it, right?

if you need an init method that does nothing, you should call it init/initialize or whatever and tell the user that he MUST call this method to make sure the class is loaded.